src/components/FormFields/DataIntegrityChecksField.js

```diff
@@ -0,0 +1,158 @@
+        }))
+        .sort((a, b) => a.label.localeCompare(b.label))
+
+    const toggle = ({ value }) => {
+        const checked = value === 'true'
+
+        if (!checked) {
+            // clear checks when "Run all" is selected
+            onChange([])
+        }
+        setRunSelected(checked)
+    }
+
+    return (
+        <FieldGroup label={i18n.t('Checks to run')}>
+            <Radio
+                name={'checksToRun'}
+                value={String(false)}
+                label={i18n.t('Run all available checks')}
+                checked={!runSelected}
+                onChange={toggle}
+            />
+            <Radio
+                name={'checksToRun'}
+                value={String(true)}
+                label={i18n.t('Only run selected checks')}
+                checked={runSelected}
+                onChange={toggle}
+            />
+            <Field
+                name={name}
+                component={ChecksTransfer}
+                options={translatedOptions}
+                label={label}
+                validate={VALIDATOR}
+                // conditional rendering of FinalForm-fields cause some issues,
+                // see https://github.com/final-form/react-final-form/issues/809
+                hidden={!runSelected}
+            />
+        </FieldGroup>
+    )
+}
+
+const LabelComponent = ({ label, severity, highlighted, disabled }) => (
+    <div
+        className={cx(styles.transferOption, {
+            [styles.highlighted]: highlighted,
+            [styles.disabled]: disabled,
+        })}
+    >
+        <div className={styles.optionName}>{label}</div>
+        <div
+            className={cx(styles.optionSeverity, {
+                [styles.highlighted]: highlighted,
+            })}
+        >{`${i18n.t('Severity')}: ${severity}`}</div>
+    </div>
+)
+
+LabelComponent.propTypes = TransferOption.propTypes
+
+const renderOption = option => (
+    <TransferOption {...option} label={<LabelComponent {...option} />} />
+)
+
+const ChecksTransfer = ({ input, meta, options = [], hidden }) => {
+    const { onChange } = input
+
+    const handleChange = useCallback(
+        ({ selected }) => {
+            onChange(selected)
+        },
+        [onChange]
+    )
+
+    if (hidden) {
+        return null
+    }
+
+    const isErr = meta.touched && meta.invalid
+
+    return (
+        <>
+            <Transfer
+                options={options}
+                onChange={handleChange}
+                selected={input.value || []}
+                renderOption={renderOption}
+                maxSelections={Infinity}
+                enableOrderChange={true}
+                filterable={true}
+                height={'450px'}
+                selectedEmptyComponent={<SelectedEmptyComponent />}
+                className={styles.transfer}
+            />
+            {isErr && <Help error={isErr}>{meta.error}</Help>}
+        </>
+    )
+}
+
+ChecksTransfer.propTypes = InputFieldFF.propTypes
```

Comment:
This will prevent eslint from linting the proptypes. It'll only lint them if they're defined directly, last I checked.